package.json

```diff
@@ -16,9 +18,13 @@
   ],
   "author": "Marshall Farrier",
   "license": "MIT",
-  "readmeFilename": "README.md",
-  "gitHead": "2c11de9333ca6ba920bd670ad637fdf50f99f323",
-  "devDependencies": {
-    "grunt": "~0.4.5"
+  "files": [
+    "LICENSE",
+    "README.md",
+    "lib/"
+  ],
+  "dependencies": {
+    "chai": "^2.2.0",
+    "mocha": "^2.2.4"
```

Comment:
The only change here that gives me pause is the hat semvar. I think we should probably just stick to patches opposed to minor updates (~ instead of ^). According to [npm semvar](https://github.com/npm/node-semver#tilde-ranges-123-12-1), the hat may constitute breaking changes (but not always).